### DISCUSSION TITLE
Can we add todo checkmark fields

### DISCUSSION THREAD

**Discussion by leasemagnets** [yoopta-editor/Yoopta-Editor#16] [Ideas]
One thing that I noticed that Yopta seems to be missing is checkmark fields?

Is there any way we can add that so it would allow us to manage todo fields as a team. This seesm to me the only real thing as a component that yopta seems to be missing

<img width="1307" alt="image" src="https://user-images.githubusercontent.com/66034432/218288976-fba32db4-29af-48f5-b66c-0323ccf0abde.png">

**Comment by Darginec05** [comment]
Hi @leasemagnets 👋

It's currently not possible to add custom components.
But I have plans for v2 to create a plugin system to add the ability to add custom components with custom rendering.

So, it will be list of packages and I have plans add Checkmark component in this list by default

**Comment by leasemagnets** [comment]
That is awesome @Darginec05 ! When do you think we will be able to add the checkmark component? I am really excited to use this to list out my tasks <3 ?

**Reply by Darginec05** [reply]
@leasemagnets I have plans launch v2 in the next month.
And Checkmark component should be in this release

**Reply by Darginec05** [reply]
Hi @leasemagnets!
Sorry, I forgot ping you about Todo plugin :) 

It's ready and you can check it here - https://yopta-editor.vercel.app/examples/withBasicExample

**Reply by maZahaca** [reply]
Recently, the url has changed. This is the actual one https://yoopta-editor.vercel.app/examples/withBaseFullSetup
